tests/sentry/incidents/test_logic.py

```diff
@@ -852,6 +853,69 @@ def test_update_from_include_all(self):
             sub.project for sub in QuerySubscription.objects.filter(alert_rules=alert_rule)
         ] == [new_project]
 
+    @pytest.mark.chris
```

Comment:
What's this for?

Reply:
It's leftover from development - it's a simple way to mark tests...then I can simply pytest -m chris to run all the tests marked as `chris`. Was a better alternative than naming tests with _meow :-)